client/zh_cn.editable.json

```
@@ -1,231 +1,231 @@
+    "projectelo.text.profile.matches": "比赛场数",
     "projectelo.text.profile.win": "胜",
     "projectelo.text.profile.loss": "负",
     "projectelo.text.profile.winrates": "胜率",
     "projectelo.text.profile.season": "赛季",
-    "projectelo.text.profile.ranked_statistics": "排位数据",
-    "projectelo.text.profile.total_played": "比赛 (总计 / 赛季)",
-    "projectelo.text.profile.highest_winstreak": "连胜 (最佳 / 当前)",
-    "projectelo.text.profile.best_elo_rate": "段位分 (最佳 / 上个赛季)",
+    "projectelo.text.profile.ranked_statistics": "排位赛数据",
+    "projectelo.text.profile.total_played": "比赛场数 (总计 / 赛季)",
+    "projectelo.text.profile.highest_winstreak": "连胜局数 (最佳 / 当前)",
+    "projectelo.text.profile.best_elo_rate": "排位赛积分 (最佳 / 上个赛季)",
     "projectelo.text.profile.best_record_time": "个人最佳记录",
     "projectelo.text.profile.muted": "屏蔽",
     "projectelo.text.profile.chat_disabled": "已禁用聊天",
-    "projectelo.text.match.win": "胜",
-    "projectelo.text.match.lose": "负",
-    "projectelo.text.match.draw": "平",
-    "projectelo.text.match.decay": "掉分",
-    "projectelo.text.match.forfeited": "弃权",
-    "projectelo.text.match.winner": "赢家",
+    "projectelo.text.match.win": "获胜",
+    "projectelo.text.match.lose": "失败",
+    "projectelo.text.match.draw": "平局",
+    "projectelo.text.match.decay": "排名分自动下降",
+    "projectelo.text.match.forfeited": "放弃比赛",
+    "projectelo.text.match.winner": "胜者",
     "projectelo.text.match.seed": "种子",
     "projectelo.text.match.final_time": "最终时间",
     "projectelo.text.match.match_players": "比赛玩家",
     "projectelo.text.match.current_match": "当前比赛",
-    "projectelo.text.achievement_badges": "进度徽章",
+    "projectelo.text.achievement_badges": "成就奖章",
     "projectelo.text.match_date": "比赛日期",
     "projectelo.text.seed_type": "种子类型",
-    "projectelo.text.open_live_replay_settings": "打开即时回放设置",
-    "projectelo.text.cinematic_camera": "旁观者视角",
-    "projectelo.text.toggle_cinematic_camera": "切换旁观者视角",
-    "projectelo.text.pause_replay": "Pause Replay",
-    "projectelo.text.resume_replay": "Resume Replay",
-    "projectelo.text.back_live_replay": "Return to Live Replay",
-    "projectelo.text.move_back_replay": "Move Back %s Seconds",
-    "projectelo.text.move_forward_replay": "Move Forward %s Seconds",
+    "projectelo.text.open_live_replay_settings": "打开实时回放设置",
+    "projectelo.text.cinematic_camera": "观察者视角",
+    "projectelo.text.toggle_cinematic_camera": "切换观察者视角",
+    "projectelo.text.pause_replay": "暂停回放",
+    "projectelo.text.resume_replay": "继续回放",
+    "projectelo.text.back_live_replay": "回到实时回放",
+    "projectelo.text.move_back_replay": "回退 %s 秒",
+    "projectelo.text.move_forward_replay": "快进 %s 秒",
     "projectelo.text.change_replay_target": "更改回放目标",
     "projectelo.text.change_replay_name": "更改回放名称",
-    "projectelo.text.open_replay_folder": "打开回放文件夹",
-    "projectelo.text.archived": "已封存",
+    "projectelo.text.open_replay_folder": "打开回放存放文件夹",
+    "projectelo.text.archived": "已打包",
```

```suggestion
    "projectelo.text.archived": "已固定",
```
zh-cn: 问题：”打包“指代不明。MCSR 的 archive 实际上是指让某些回放不会随着达到上限而自动删除。